kano_updater/ui/media/flappy-judoka/src/graphics/hud.py

```diff
@@ -52,11 +52,11 @@ def __init__(self, display):
         self.font_path = os.path.join(fonts_path, 'zx_spectrum-7_bold.ttf')
         self.setup_fonts()
 
-        self.controls_hint_text = self.hint_renderer.render('[SPACE] or [UP] to Jump', 1, self.WHITE)
+        self.controls_hint_text = self.hint_renderer.render(_("[SPACE] or [UP] to Jump"), 1, self.WHITE)
         self.controls_hint_text_x = self.display_width * self.CONTROLS_HINT_X
         self.controls_hint_text_y = self.display_height * self.CONTROLS_HINT_Y
 
-        self.quit_hint_text = self.hint_renderer.render('[ESC] or [Q] to Quit', 1, self.WHITE)
+        self.quit_hint_text = self.hint_renderer.render(_("[ESC] or [Q] to Quit"), 1, self.WHITE)
```

Comment:
Not sure if I've spotted where Flappy is "installing" `kano-i18n` and the translation functions. Do you see the strings with the wide font under `en_QQ`?